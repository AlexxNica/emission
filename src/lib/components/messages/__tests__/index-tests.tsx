import * as React from "react"
import "react-native"
import * as renderer from "react-test-renderer"
import Inbox from "../inbox/inbox"

it("Sets up the right view hierarchy", () => {
    const tree = renderer.create(<Inbox />).toJSON()
    expect(tree).toMatchSnapshot()
})
