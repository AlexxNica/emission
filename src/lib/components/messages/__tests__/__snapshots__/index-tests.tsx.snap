// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Looks correct when rendered 1`] = `
<RCTScrollView>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "black",
            "fontSize": 30,
            "paddingLeft": 20,
            "paddingRight": 20,
            "textAlign": "left",
          },
          Object {
            "marginTop": 10,
          },
          Object {
            "fontFamily": "AGaramondPro-Regular",
          },
        ]
      }
    >
      Messages
    </Text>
    <View
      style={
        Object {
          "marginLeft": 20,
          "marginRight": 20,
          "marginTop": 20,
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <AROpaqueImageView
          imageURL="https://d32dm0rphc51dk.cloudfront.net/rmpESHC2mhtxdTq0S6mMCw/tall.jpg"
          style={
            Object {
              "height": 58,
              "width": 58,
            }
          }
        />
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "marginLeft": 15,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 11,
                    "textAlign": "left",
                  },
                  Object {},
                  Object {
                    "fontFamily": "Avant Garde Gothic ITCW01Dm",
                  },
                ]
              }
            >
              PATRICK PARRISH
            </Text>
            <View
              style={
                Object {
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "flex-end",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Array [
                    Object {
                      "color": "#cccccc",
                      "fontSize": 11,
                      "textAlign": "left",
                    },
                    Object {},
                    Object {
                      "fontFamily": "Avant Garde Gothic ITCW01Dm",
                    },
                  ]
                }
              >
                11:00 AM
              </Text>
              <View
                style={
                  Object {
                    "backgroundColor": "#6e1fff",
                    "borderRadius": 4,
                    "height": 8,
                    "marginLeft": 4,
                    "marginVertical": 3,
                    "width": 8,
                  }
                }
              />
            </View>
          </View>
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 14,
                  },
                  Object {},
                  Object {
                    "fontFamily": "AGaramondPro-Italic",
                  },
                ]
              }
            >
              Ian Stell · 
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 14,
                  },
                  Object {},
                  Object {
                    "fontFamily": "AGaramondPro-Italic",
                  },
                ]
              }
            >
              Threestool, 2014
            </Text>
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "color": "black",
                  "fontSize": 15,
                  "textAlign": "left",
                },
                Object {},
                Object {
                  "fontFamily": "AGaramondPro-Regular",
                },
              ]
            }
          >
            Hi Katarina, thanks for for reaching out with regards to this artwork
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#e5e5e5",
            "height": 1,
            "marginTop": 20,
            "width": "100%",
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "marginLeft": 20,
          "marginRight": 20,
          "marginTop": 20,
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <AROpaqueImageView
          imageURL="https://d32dm0rphc51dk.cloudfront.net/rmpESHC2mhtxdTq0S6mMCw/tall.jpg"
          style={
            Object {
              "height": 58,
              "width": 58,
            }
          }
        />
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "marginLeft": 15,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 11,
                    "textAlign": "left",
                  },
                  Object {},
                  Object {
                    "fontFamily": "Avant Garde Gothic ITCW01Dm",
                  },
                ]
              }
            >
              PATRICK PARRISH
            </Text>
            <View
              style={
                Object {
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "flex-end",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Array [
                    Object {
                      "color": "#cccccc",
                      "fontSize": 11,
                      "textAlign": "left",
                    },
                    Object {},
                    Object {
                      "fontFamily": "Avant Garde Gothic ITCW01Dm",
                    },
                  ]
                }
              >
                11:00 AM
              </Text>
              <View
                style={
                  Object {
                    "backgroundColor": "#6e1fff",
                    "borderRadius": 4,
                    "height": 8,
                    "marginLeft": 4,
                    "marginVertical": 3,
                    "width": 8,
                  }
                }
              />
            </View>
          </View>
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 14,
                  },
                  Object {},
                  Object {
                    "fontFamily": "AGaramondPro-Italic",
                  },
                ]
              }
            >
              Ian Stell · 
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 14,
                  },
                  Object {},
                  Object {
                    "fontFamily": "AGaramondPro-Italic",
                  },
                ]
              }
            >
              Threestool, 2014
            </Text>
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "color": "black",
                  "fontSize": 15,
                  "textAlign": "left",
                },
                Object {},
                Object {
                  "fontFamily": "AGaramondPro-Regular",
                },
              ]
            }
          >
            Hi Katarina, thanks for for reaching out with regards to this artwork
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#e5e5e5",
            "height": 1,
            "marginTop": 20,
            "width": "100%",
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "marginLeft": 20,
          "marginRight": 20,
          "marginTop": 20,
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <AROpaqueImageView
          imageURL="https://d32dm0rphc51dk.cloudfront.net/rmpESHC2mhtxdTq0S6mMCw/tall.jpg"
          style={
            Object {
              "height": 58,
              "width": 58,
            }
          }
        />
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "marginLeft": 15,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 11,
                    "textAlign": "left",
                  },
                  Object {},
                  Object {
                    "fontFamily": "Avant Garde Gothic ITCW01Dm",
                  },
                ]
              }
            >
              PATRICK PARRISH
            </Text>
            <View
              style={
                Object {
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "flex-end",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Array [
                    Object {
                      "color": "#cccccc",
                      "fontSize": 11,
                      "textAlign": "left",
                    },
                    Object {},
                    Object {
                      "fontFamily": "Avant Garde Gothic ITCW01Dm",
                    },
                  ]
                }
              >
                11:00 AM
              </Text>
              <View
                style={
                  Object {
                    "backgroundColor": "#6e1fff",
                    "borderRadius": 4,
                    "height": 8,
                    "marginLeft": 4,
                    "marginVertical": 3,
                    "width": 8,
                  }
                }
              />
            </View>
          </View>
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 14,
                  },
                  Object {},
                  Object {
                    "fontFamily": "AGaramondPro-Italic",
                  },
                ]
              }
            >
              Ian Stell · 
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 14,
                  },
                  Object {},
                  Object {
                    "fontFamily": "AGaramondPro-Italic",
                  },
                ]
              }
            >
              Threestool, 2014
            </Text>
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "color": "black",
                  "fontSize": 15,
                  "textAlign": "left",
                },
                Object {},
                Object {
                  "fontFamily": "AGaramondPro-Regular",
                },
              ]
            }
          >
            Hi Katarina, thanks for for reaching out with regards to this artwork
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#e5e5e5",
            "height": 1,
            "marginTop": 20,
            "width": "100%",
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "marginLeft": 20,
          "marginRight": 20,
          "marginTop": 20,
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <AROpaqueImageView
          imageURL="https://d32dm0rphc51dk.cloudfront.net/rmpESHC2mhtxdTq0S6mMCw/tall.jpg"
          style={
            Object {
              "height": 58,
              "width": 58,
            }
          }
        />
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "marginLeft": 15,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 11,
                    "textAlign": "left",
                  },
                  Object {},
                  Object {
                    "fontFamily": "Avant Garde Gothic ITCW01Dm",
                  },
                ]
              }
            >
              PATRICK PARRISH
            </Text>
            <View
              style={
                Object {
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "flex-end",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Array [
                    Object {
                      "color": "#cccccc",
                      "fontSize": 11,
                      "textAlign": "left",
                    },
                    Object {},
                    Object {
                      "fontFamily": "Avant Garde Gothic ITCW01Dm",
                    },
                  ]
                }
              >
                11:00 AM
              </Text>
              <View
                style={
                  Object {
                    "backgroundColor": "#6e1fff",
                    "borderRadius": 4,
                    "height": 8,
                    "marginLeft": 4,
                    "marginVertical": 3,
                    "width": 8,
                  }
                }
              />
            </View>
          </View>
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 14,
                  },
                  Object {},
                  Object {
                    "fontFamily": "AGaramondPro-Italic",
                  },
                ]
              }
            >
              Ian Stell · 
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 14,
                  },
                  Object {},
                  Object {
                    "fontFamily": "AGaramondPro-Italic",
                  },
                ]
              }
            >
              Threestool, 2014
            </Text>
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "color": "black",
                  "fontSize": 15,
                  "textAlign": "left",
                },
                Object {},
                Object {
                  "fontFamily": "AGaramondPro-Regular",
                },
              ]
            }
          >
            Hi Katarina, thanks for for reaching out with regards to this artwork
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#e5e5e5",
            "height": 1,
            "marginTop": 20,
            "width": "100%",
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "marginLeft": 20,
          "marginRight": 20,
          "marginTop": 20,
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <AROpaqueImageView
          imageURL="https://d32dm0rphc51dk.cloudfront.net/rmpESHC2mhtxdTq0S6mMCw/tall.jpg"
          style={
            Object {
              "height": 58,
              "width": 58,
            }
          }
        />
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "column",
              "marginLeft": 15,
            }
          }
        >
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 11,
                    "textAlign": "left",
                  },
                  Object {},
                  Object {
                    "fontFamily": "Avant Garde Gothic ITCW01Dm",
                  },
                ]
              }
            >
              PATRICK PARRISH
            </Text>
            <View
              style={
                Object {
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "flex-end",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Array [
                    Object {
                      "color": "#cccccc",
                      "fontSize": 11,
                      "textAlign": "left",
                    },
                    Object {},
                    Object {
                      "fontFamily": "Avant Garde Gothic ITCW01Dm",
                    },
                  ]
                }
              >
                11:00 AM
              </Text>
              <View
                style={
                  Object {
                    "backgroundColor": "#6e1fff",
                    "borderRadius": 4,
                    "height": 8,
                    "marginLeft": 4,
                    "marginVertical": 3,
                    "width": 8,
                  }
                }
              />
            </View>
          </View>
          <View
            style={
              Object {
                "flex": 1,
                "flexDirection": "row",
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 14,
                  },
                  Object {},
                  Object {
                    "fontFamily": "AGaramondPro-Italic",
                  },
                ]
              }
            >
              Ian Stell · 
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 14,
                  },
                  Object {},
                  Object {
                    "fontFamily": "AGaramondPro-Italic",
                  },
                ]
              }
            >
              Threestool, 2014
            </Text>
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            numberOfLines={1}
            style={
              Array [
                Object {
                  "color": "black",
                  "fontSize": 15,
                  "textAlign": "left",
                },
                Object {},
                Object {
                  "fontFamily": "AGaramondPro-Regular",
                },
              ]
            }
          >
            Hi Katarina, thanks for for reaching out with regards to this artwork
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#e5e5e5",
            "height": 1,
            "marginTop": 20,
            "width": "100%",
          }
        }
      />
    </View>
  </View>
</RCTScrollView>
`;
